src/imagetilesource.js

```diff
@@ -50,6 +50,7 @@
      * @extends OpenSeadragon.TileSource
      * @param {Object} options Options object.
      * @param {String} options.url URL of the image
+     * @param {String} options.referenceStripThumbnailUrl URL of the dedicated thumbnail image
```

Comment:
OK. I think it makes sense now...sort of ;)

My problem was rooted in the impression that ReferenceStrip works only for "image" tile sources. I don't know where I got that from, but when this false premise is removed, your comments are clear.